components/async-status-mgt/org.wso2.carbon.identity.framework.async.status.mgt/src/main/java/org/wso2/carbon/identity/framework/async/status/mgt/util/Utils.java

```diff
@@ -0,0 +1,73 @@
+package org.wso2.carbon.identity.framework.async.status.mgt.util;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.wso2.carbon.database.utils.jdbc.NamedJdbcTemplate;
+import org.wso2.carbon.database.utils.jdbc.exceptions.DataAccessException;
+import org.wso2.carbon.identity.framework.async.status.mgt.constant.AsyncStatusMgtConstants;
+import org.wso2.carbon.identity.framework.async.status.mgt.exception.AsyncStatusMgtClientException;
+import org.wso2.carbon.identity.framework.async.status.mgt.exception.AsyncStatusMgtServerException;
+import org.wso2.carbon.identity.framework.async.status.mgt.internal.AsyncStatusMgtDataHolder;
+
+import javax.sql.DataSource;
+
+public class Utils {
```

Comment:
Let's do not have common util class.
Split into separate util classes based on the functionality.